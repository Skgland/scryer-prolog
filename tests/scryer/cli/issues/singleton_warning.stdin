['tests-pl/issue812-singleton-warning.pl'].
halt.
