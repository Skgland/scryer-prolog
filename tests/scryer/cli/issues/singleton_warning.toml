# issue 812
args = ["-f", "--no-add-history"]
binary = true
