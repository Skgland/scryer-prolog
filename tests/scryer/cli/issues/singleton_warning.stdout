Warning: singleton variables X at line 4 of issue812-singleton-warning.pl
   true.
